variables.tf

```
@@ -407,3 +437,9 @@ variable "terminated_instances" {
   type        = string
   default     = "30"
 }
+
+variable "tracking_policy_metric" {
+  description = "Enter Target Tracking Policy's metric name. Allowed Values are: AvgCPUUtilization, AvgNetworkIn, AvgNetworkOut, ALBRequestCount"
+  type        = string
+  default     = "AvgCPUUtilization"
+}
```

I would update this list to the exact values required, again in an effort to unify the 4 resources into a single one.
```suggestion
variable "tracking_policy_metric" {
  description = "Enter Target Tracking Policy's metric name. Allowed Values are: ASGAverageCPUUtilization, ASGAverageNetworkIn, ASGAverageNetworkOut, ALBRequestCountPerTarget"
  type        = string
  default     = "ASGAverageCPUUtilization"
}
```

---

I have updated the variables with the exact values required